libs/SmartStore/src/com/salesforce/androidsdk/smartstore/app/SmartStoreSDKManager.java

```diff
@@ -424,4 +422,39 @@ public void onSelected() {
         ));
         return devSupportInfos;
     }
+
+    /**
+     * Get key value store with given name for current user
+     * @param storeName store name
+     * @return a KeyValueEncryptedFileStore
+     */
+    public KeyValueEncryptedFileStore getKeyValueStore(String storeName) {
```

Comment:
Convenience methods to get named key value stores for current user / some user / some user + community.
Do we need to have global key value stores also?

Not done: the methods to get all key value stores of a user / to clean them up.

Reply:
If it's not too much effort, it would be nice to add the "global" option as well. Keeps the feature set at parity with everything else in `SmartStore`.